### PR TITLE
feat: [PL-23997]: Quick filter (exclude events) in Audit trails

### PR DESCRIPTION
**Pull request by avinashmadhwani02**
Screenshot 2022-03-24 at 11 25 21 AM

### CODE REVIEW

**Comment by avinashmadhwani02** [comment]
dockerize

**Comment by avinashmadhwani02** [comment]
dockerize

File: src/modules/18-audit-trail/pages/AuditTrails/AuditTrailsPage.tsx [hunk 1]
```diff
@@ -98,15 +108,35 @@ const AuditTrailsPage: React.FC = () => {
...
+                `${selectedDates[0].toLocaleDateString()} - ${selectedDates[1].toLocaleDateString()}`
+              }
+            />
+            <DropDown
+              items={getShowEventsDropdownList()}
+              filterable={false}
+              addClearBtn={true}
+              placeholder={getString('all')}
+              value={selectedFilterProperties?.staticFilter}
+              width={170}
```
Comment (mayankVermaHarness) [inline src/modules/18-audit-trail/pages/AuditTrails/AuditTrailsPage.tsx:127]: Any other way to add this width?

**Review by mayankVermaHarness — APPROVED** [review]

File: src/modules/18-audit-trail/pages/AuditTrails/AuditTrailsPage.tsx (hunk 1 above)
Comment (avinashmadhwani02) [reply src/modules/18-audit-trail/pages/AuditTrails/AuditTrailsPage.tsx:127]: No,

Ideally we should have variants of Select Component as per our UX, but right now we don't have a support for it

**Review by mayankVermaHarness — DISMISSED** [review]

**Review by kapilkumawat86 — APPROVED** [review]

**Review by mayankVermaHarness — APPROVED** [review]

**Review by harshilbits — APPROVED** [review]

**Pull request merged**